src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -99,7 +98,7 @@ object WorkflowDescriptor {
   }
 
   def apply(id: WorkflowId, sourceFiles: WorkflowSourceFiles, conf: Config): WorkflowDescriptor = {
-    validateWorkflowDescriptor(id, sourceFiles, CromwellServer.backend, conf) match {
```

Comment:
I noticed that WorkflowManagerSystem (extended by CromwellServer) still defines "backend" but it's only ever used by tests. And I'm pretty sure they aren't *really* using it.